mlir/docs/DeclarativeRewrites.md

```diff
@@ -380,6 +380,11 @@ template. The string can be an arbitrary C++ expression that evaluates into some
 C++ object expected at the `NativeCodeCall` site (here it would be expecting an
 array attribute). Typically the string should be a function call.
 
+In the case of properties, the return value of the `NativeCodeCall` should
+be in terms of the _interface_ type of a property. For example, the `NativeCodeCall`
```

Comment:
Could you perhaps link to mlir/include/mlir/IR/Properties.td ? (it makes clearer origin of interface and where to look it up for others).